docs/releases/unreleased.md

```diff
@@ -10,7 +10,7 @@ This release adds support for Kubernetes version vTBD.
 
 ## New features 🌟
```

Comment:
This PR is introducing a breaking change on the etcd upgrade process (and in the certs renewal process?). This should be noted in the release notes.